pom.xml

```diff
@@ -163,7 +163,7 @@
                             <goal>exec</goal>
                         </goals>
                         <configuration>
-                            <skip>false</skip>
+                            <skip>true</skip>
```

Comment:
This bat files copies the compiled jar to my dev server. You can find it in the scripts folder.